src/chat_participants/dataPilot.ts

```
@@ -0,0 +1,188 @@
+        "Error registering mcp client",
+        error,
+      );
+      throw error;
+    }
+  }
+  private async handleRequest(
+    request: vscode.ChatRequest,
+    context: vscode.ChatContext,
+    stream: vscode.ChatResponseStream,
+    token: vscode.CancellationToken,
+  ): Promise<vscode.ChatResult | void> {
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)